NOTICE

```diff
@@ -0,0 +1,697 @@
+CSP
+Copyright 2023-2024 Point72, L.P.
+
+
+
+This product includes software from https://github.com/abseil/abseil-cpp (Apache 2.0)
+ * Copyright (c) The abseil authors
+
+This product includes software from https://github.com/google/brotli/ (BSD 3-clause)
```

Comment:
Brotli is MIT https://github.com/google/brotli/blob/master/LICENSE

Reply:
doing it a better way now